src/qs_linres_types.F

```diff
@@ -58,6 +58,7 @@ MODULE qs_linres_types
       INTEGER                                   :: restart_every
       REAL(KIND=dp)                             :: energy_gap
       INTEGER                                   :: max_iter
+      INTEGER                                   :: every_n_step
```

Comment:
It seems you only need the value right after you read it from the input. Hence, there is no need to store it in `qs_linres_types`. Please remove it.

Reply:
Added it in the derived type in case it could be useful for other tasks in the future.